dbt_semantic_interfaces/naming/dundered.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+from dbt_semantic_interfaces.references import EntityReference
+from dbt_semantic_interfaces.type_enums.time_granularity import TimeGranularity
+
+DUNDER = "__"
```

Comment:
Instead of defining this again here, shouldn't we import the `DUNDER` defined in `keywords.py`?

Reply:
Refactor error - updated.